clickhouse/columns/tuple.h

```diff
@@ -22,7 +22,7 @@ class ColumnTuple : public Column {
 
 public:
     /// Appends content of given column to the end of current one.
-    void Append(ColumnRef) override { }
+    void Append(ColumnRef column) override;
```

Comment:
Please add some unit tests that validate implementation of `ColumnTuple::Append()`, both positive and negative cases.